vllm/model_executor/layers/attention.py

```diff
@@ -160,6 +160,8 @@ def forward(
                 attn_bias=input_metadata.attn_bias,
                 p=0.0,
                 scale=self.scale,
+                op=xops.fmha.MemoryEfficientAttentionFlashAttentionOp[0] if
+                (torch.cuda.is_available() and torch.version.hip) else None,
```

Comment:
BTW, why do we need `torch.cuda.is_available` here and in other places? In which case can this be False?